### PR TITLE
Small to edit to worst fans method, now passes in spec harness

### PR DESCRIPTION
**Pull request by margoflewelling**
Small edit to the method, Now passes in the spec harness

### CODE REVIEW

File: lib/modules/league_statistics.rb [hunk 1]
```diff
@@ -137,15 +137,15 @@ def home_win_percentage(team_id)
...
+    percent.round(2)
   end
 
   def away_win_percentage(team_id)
     away_wins = away_games(team_id).find_all do |game|
       game.result == "WIN"
     end
-    percent(away_wins.length, home_games(team_id).length.to_f)
-    percent = (away_wins.length / home_games(team_id).length.to_f) * 100
+    percent = (away_wins.length / away_games(team_id).length.to_f) * 100
```
Comment (kelshadarby) [inline lib/modules/league_statistics.rb:148]: Nice catch!

File: lib/modules/league_statistics.rb [hunk 2]
```diff
@@ -137,15 +137,15 @@ def home_win_percentage(team_id)
     home_wins = home_games(team_id).find_all do |game|
       game.result == "WIN"
     end
-    percent(home_wins.length, home_games(team_id).length.to_f)
+    percent = (home_wins.length / home_games(team_id).length.to_f) * 100
+    percent.round(2)
```
Comment (kelshadarby) [inline lib/modules/league_statistics.rb:141]: Good work!

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -205,7 +205,7 @@ def test_getting_the_team_with_the_best_fans
   end
 
   def test_getting_the_teams_with_the_worst_fans
-    @stat_tracker.stubs(:away_win_percentage).returns(80)
```
Comment (kelshadarby) [inline test/stat_tracker_test.rb:208]: Great use of stubs!

File: lib/modules/league_statistics.rb (hunk 1 above)
Comment (chrisstankus99) [inline lib/modules/league_statistics.rb:148]: Good job!

**Pull request merged**